testing/test_pytester.py

```diff
@@ -24,51 +24,52 @@ def test_make_hook_recorder(testdir) -> None:
     assert not recorder.getfailures()
 
     pytest.xfail("internal reportrecorder tests need refactoring")
-
-    class rep:
-        excinfo = None
-        passed = False
-        failed = True
-        skipped = False
-        when = "call"
-
-    recorder.hook.pytest_runtest_logreport(report=rep)
-    failures = recorder.getfailures()
-    assert failures == [rep]
-    failures = recorder.getfailures()
-    assert failures == [rep]
-
-    class rep2:
-        excinfo = None
-        passed = False
-        failed = False
-        skipped = True
-        when = "call"
-
-    rep2.passed = False
-    rep2.skipped = True
-    recorder.hook.pytest_runtest_logreport(report=rep2)
-
-    modcol = testdir.getmodulecol("")
-    rep3 = modcol.config.hook.pytest_make_collect_report(collector=modcol)
-    rep3.passed = False
-    rep3.failed = True
-    rep3.skipped = False
-    recorder.hook.pytest_collectreport(report=rep3)
-
-    passed, skipped, failed = recorder.listoutcomes()
-    assert not passed and skipped and failed
-
-    numpassed, numskipped, numfailed = recorder.countoutcomes()
-    assert numpassed == 0
-    assert numskipped == 1
-    assert numfailed == 1
-    assert len(recorder.getfailedcollections()) == 1
-
-    recorder.unregister()
-    recorder.clear()
-    recorder.hook.pytest_runtest_logreport(report=rep3)
-    pytest.raises(ValueError, recorder.getfailures)
+    if False:  # pragma: no cover
```

Comment:
I'd rather leave the code unchanged and ignore the hint from codecov that this is uncovered.
(this avoids unnecessary diff churn etc)

Reply:
OK, removed. But from my understanding, because of how the coverage check works, whenever the file is touched, it will fail.

Reply:
It "fails" when this part is contained in a diff.